### DISCUSSION TITLE
Suggestions for filter lists’ update

### DISCUSSION THREAD

**Discussion by DimitrisKalaboukas** [uBlockOrigin/uAssets#26325] [Ideas]
I think that you should remove Fanboy – Anti-Facebook list, because it isn’t actively maintained and it causes a lot of breakages, which aren’t fixed even if they have been reported long time ago.

Also, you could add [Scam Blocklist by DurableNapkin](https://raw.githubusercontent.com/durablenapkin/scamblocklist/master/adguard.txt) in “Malicious domains” section, [AdGuard DNS filter](https://adguardteam.github.io/AdGuardSDNSFilter/Filters/filter.txt) in “Multiple uses” section, and [Dandelion Sprout's Annoyances List](https://raw.githubusercontent.com/DandelionSprout/adfilt/master/AnnoyancesList) in “Annoyances” section.

**Comment by stephenhawk8054** [comment]
> a lot of breakages

What breakages?

**Reply by DimitrisKalaboukas** [reply]
E.g. https://github.com/ryanbr/fanboy-adblock/issues/1551

**Reply by stephenhawk8054** [reply]
I mean...

It's literally a video from **facebook**, the video loads in an iframe with the source link `https://www.facebook.com/v2.5/plugins/video.php`. The list is literally called **Anti-Facebook**. It's doing the job correctly. It's not breakage.

Imagine another user who wants to block all **facebook** connections, enables that list (because that list is called **Anti-Facebook**) and suddenly when they visit that page, a literal **facebook** video loads in their browser, just because another user reported that the video is "useful"?

If you want to load **facebook** videos, you should not enable that list. It's a specialized list. It's not a matter of "useful" or not. The blocking is literally its intended feature. If you want to whitelist at some pages, you can whitelist it yourself, not report to the list to exclude that site for you.
